packages/document-grounding/src/tests/pipelines-api.test.ts

```diff
@@ -4,7 +4,7 @@ import {
   mockClientCredentialsGrantCall
 } from '../../../../test-util/mock-http.js';
 import { PipelinesApi } from '../client/api/index.js';
-import type { Pipelines } from '../client/api/index.js';
+import type { GetPipelines } from '../client/api/index.js';
```

Comment:
Is this something that was missed before?

Reply:
yes, and the other type import. Somehow the tests passed before